swagger.yaml

```diff
@@ -294,7 +558,7 @@ paths:
                   $ref: '#/definitions/FiatAccountInfoResponse'
       security:
         - siwe_auth: []
-  /accounts/{fiatAccountId}:
+  /accounts/delete/{fiatAccountId}:
```

Comment:
Wondering if this ambiguity warrants further discussion around `POST /accounts/{fiatAccountSchema}`.  I would actually expect that endpoint to take all parameters in the body (vs some in the body and some in the path).  Is there a reason to use both path and body for that POST endpoint @jophish @jh2oman @cajubelt ?

Reply:
all in the body does seem a bit cleaner

Reply:
+1 to having `fiatAccountSchema` in the body and just doing `/accounts/{fiatAccountId}` here

Reply:
+1 all in the body feels cleaner to me as well.  I don't think we have `fiatAccountId` on the request to add new fiat account

Reply:
Yeah, I believe it would just be `POST /accounts` with the request body being e.g:

```
{
  fiatAccountSchema: ...
  data: {
   ...
  }
}
```

or, something like that.